src/coreclr/inc/clrconfigvalues.h

```diff
@@ -707,6 +707,7 @@ RETAIL_CONFIG_STRING_INFO(INTERNAL_EventPipeConfig, W("EventPipeConfig"), "Confi
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeRundown, W("EventPipeRundown"), 1, "Enable/disable eventpipe rundown.")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeCircularMB, W("EventPipeCircularMB"), 1024, "The EventPipe circular buffer size in megabytes.")
 RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeProcNumbers, W("EventPipeProcNumbers"), 0, "Enable/disable capturing processor numbers in EventPipe event headers")
+RETAIL_CONFIG_DWORD_INFO(INTERNAL_EventPipeOutputStreaming, W("EventPipeOutputStreaming"), 0, "Enable/disable streaming for trace file set in COMPlus_EventPipeOutputPath.  Non-zero values enable streaming.")
```

Comment:
OK, since it where belonged to the same group of properties I put them it together with the rest, but this can of course be adjusted to use DOTNET_.

Reply:
If we switch I'd recommend we switch the whole group together rather than have a mixture of COMPlus and DOTNET vars. Its probably not worth trying to put in scope of the current PR.

Reply:
Agree, I suggest we keep this as is and then make a change at a later point in time, moving the whole set over to `DOTNET_` prefix.